src/App.jsx

```diff
@@ -14,57 +15,118 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState(goods[8]);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {selectedGood === 0 ? (
```

Comment:
The condition `selectedGood === 0` is incorrect for checking if no goods are selected. Since `selectedGood` is initialized with `goods[8]` (a string), consider using `selectedGood === ''` or another appropriate default value to represent no selection.